main.tf

```diff
@@ -194,12 +194,12 @@ resource "castai_autoscaler" "castai_autoscaler_policies" {
   depends_on = [helm_release.castai_agent, helm_release.castai_evictor]
 }
 
-resource "helm_release" "castai_sec_agent" {
```

Comment:
We don't want to rename variables `install_security_agent` and `sec_agent_values`?